### ISSUE TITLE
Swift Compiler Error due to multiple Ambiguous use of 'validate' error

### ISSUE THREAD

**Issue by code-draft** [adamwaite/Validator#128]
After updating to **Xcode Version 11.0 (11A420a)** on **macOS Mojave Version 10.14.6 (18G95)**,  with **Cocopods Version 1.7.5**, **Validator Pod Version swift-4.2**

`pod 'Validator', :git => 'https://github.com/adamwaite/Validator.git', :branch => 'swift-4.2'`

I have been facing this error.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/22313617/65481392-c4deee00-deb2-11e9-86b7-42479faa3bbe.png">

**Comment by kennybuc** [comment]
@code-draft I have got the same issue on Xcode 11

**Comment by antonioreyna** [comment]
me too, any fix?

**Comment by adamwaite** [comment]
Hi guys I'll look into this when I get a bit of spare time. I haven't had chance to run the lib on Xcode 11 yet.

**Comment by Moistbobo** [comment]
I got around this problem by renaming the validate function by adding an extra e so it wouldn't clash with what ever was causing the ambiguous errors.

I don't have and swift/objC experience as I work mostly in react native, so I don't know if this will have any ill effects. 

```
   open func validateOnInputChange(enabled: Bool) {
        switch enabled {
        case true: addTarget(self, action: #selector(UISlider.validatee), for: .valueChanged)
        case false: removeTarget(self, action: #selector(UISlider.validatee), for: .valueChanged)
        }
    }
    
    @objc private func validatee(sender: UISlider) {
        sender.validate()
    }```

**Comment by mostafakram** [comment]
Anyone is working on a PR for this?

**Comment by baha-che** [comment]
I have got the same issue on Xcode 11, any fix?

**Comment by drdaz** [comment]
Funny thing. I cloned this and was about to make a PR. But the source builds fine in master. It's been fixed, but not released:

https://github.com/adamwaite/Validator/blob/master/Validator/Sources/UIKit%2BValidator/UITextField%2BValidator.swift

**Comment by drdaz** [comment]
@adamwaite is master safe to use?

**Comment by ssferrazza** [comment]
I guess this one is the cause:
https://developer.apple.com/documentation/uikit/uiresponder/3229892-validate

Added in iOS 13

**Comment by CGseng** [comment]
I have got the same issue on Xcode 11

**Comment by piv199** [comment]
@code-draft the problem is that you are using outdated branch instead of master (where everything works as expected). Or are there any requirements to use that branch?

Just use: `pod Validator`

**Comment by FabrizioSposetti-U** [comment]
I'm using 'pod Validator' and the error still remains. Some news?

**Comment by piv199** [comment]
@FabrizioSposetti-U Have you tried `pod update Validator`? The latest version is `3.2.1`. Check your Podfile.lock.

**Comment by FabrizioSposetti-U** [comment]
Yes, it works. Now im facing troubles because of merge method. I was using it a lot and it does not exist any more.
